src/Components/test/E2ETest/ServerExecutionTests/GlobalizationTest.cs

```diff
@@ -8,29 +8,30 @@
 using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
 using Microsoft.AspNetCore.E2ETesting;
 using OpenQA.Selenium;
-using OpenQA.Selenium.Interactions;
 using OpenQA.Selenium.Support.UI;
+using TestServer;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Microsoft.AspNetCore.Components.E2ETest.ServerExecutionTests
 {
     // For now this is limited to server-side execution because we don't have the ability to set the
     // culture in client-side Blazor.
-    public class GlobalizationTest : BasicTestAppTestBase
+    public class GlobalizationTest : ServerTestBase<AspNetSiteServerFixture>
     {
         public GlobalizationTest(
             BrowserFixture browserFixture,
-            ToggleExecutionModeServerFixture<Program> serverFixture,
+            AspNetSiteServerFixture serverFixture,
             ITestOutputHelper output)
-            : base(browserFixture, serverFixture.WithServerExecution(), output)
+            : base(browserFixture, serverFixture, output)
         {
+            serverFixture.BuildWebHostMethod = serverFixture.BuildWebHostMethod = TestServer.Program.BuildWebHost<InternationalizationStartup>;
         }
 
         protected override void InitializeAsyncCore()
         {
             // On WebAssembly, page reloads are expensive so skip if possible
-            Navigate(ServerPathBase, _serverFixture.ExecutionMode == ExecutionMode.Client);
+            Navigate(ServerPathBase);
```

Comment:
This line of codes no longer seems to correspond with the comment above it. Can you update one of them so they match?

Also does this mean we're now always doing a full reload on WebAssembly mode? Does that make the test run substantially longer?

Reply:
Yeah, this was a bad copy pasta in the tests. These tests only run on blazor server-side